### PR TITLE
Fix hopper block

### PR DESCRIPTION
**Pull request by Creeperface01**

### CODE REVIEW

**Comment by Snake1999** [comment]
CI failed? What happened ...

**Comment by Creeperface01** [comment]
I probably badlly copied the code..

**Comment by Creeperface01** [comment]
Or idk

**Comment by imjack** [comment]
Looks fine to me. Currently they can't be broke without this fix

**Comment by Creeperface01** [comment]
They can but only in creative

File: src/main/java/cn/nukkit/block/BlockHopper.java [hunk 1]
```diff
@@ -122,4 +133,18 @@ public int onUpdate(int type) {
...
+
+    @Override
+    public int getToolType() {
+        return ItemTool.TYPE_PICKAXE;
+    }
+
+    @Override
+    public int[][] getDrops(Item item) {
+        if (item.getTier() >= ItemTool.TIER_WOODEN) {
+            return new int[][]{{Item.CAULDRON, 0, 1}};
```
Comment (imjack) [inline src/main/java/cn/nukkit/block/BlockHopper.java]: Item dropped should be a hopper

**Comment by dcmwai** [comment]
I use the CI build #2056 but see the following error...

[ALERT] java.lang.NullPointerException
        at cn.nukkit.Player.handleDataPacket(Player.java:3349)
        at cn.nukkit.network.Network.processPackets(Network.java:185)
        at cn.nukkit.network.Network.processBatch(Network.java:156)
        at cn.nukkit.Player.handleDataPacket(Player.java:1896)
        at cn.nukkit.network.RakNetInterface.handleEncapsulated(RakNetInterface.java:155)
        at cn.nukkit.raknet.server.ServerHandler.handlePacket(ServerHandler.java:123)
        at cn.nukkit.network.RakNetInterface.process(RakNetInterface.java:64)
        at cn.nukkit.network.Network.processInterfaces(Network.java:76)
        at cn.nukkit.Server.tick(Server.java:1009)
        at cn.nukkit.Server.tickProcessor(Server.java:807)
        at cn.nukkit.Server.start(Server.java:786)
        at cn.nukkit.Server.<init>(Server.java:471)
        at cn.nukkit.Nukkit.main(Nukkit.java:68)

**Comment by imjack** [comment]
@dcmwai create a issue

**Pull request merged**